apps/web/src/components/milestone-logs/MilestoneTable.tsx

```diff
@@ -1,20 +1,49 @@
-import { PageOptions, Pagination } from '../Pagination';
 import { useEffect, useState } from 'react';
-import { ApplicantStatusAuditRO, formatDate } from '@ien/common';
-import { getHumanizedDuration } from '@services';
+import dayjs from 'dayjs';
+
+import { PageOptions, Pagination } from '../Pagination';
+import { ApplicantStatusAuditRO, formatDate, IENApplicantUpdateStatusDTO } from '@ien/common';
+import { getHumanizedDuration, updateMilestone } from '@services';
 import { useApplicantContext } from '../applicant/ApplicantContext';
+import { AddMilestone, EditMilestone } from './recruitment/Milestone';
+import { useAuthContext } from '../AuthContexts';
+import editIcon from '@assets/img/edit.svg';
 
 interface MilestoneTableProps {
   parentStatus: number;
 }
 
 const DEFAULT_TAB_PAGE_SIZE = 5;
 
+const wasAddedByLoggedInUser = (loggedInId?: string | null, addedById?: string) => {
+  return loggedInId && loggedInId.toString() === addedById;
```

Comment:
Why do we need `toString()`?

Reply:
good question, I think originally I wrote a similar one and loggedInId was a type Number at the time, will fix this mistake!